app/services/dqt/trainee_update.rb

```diff
@@ -16,7 +16,10 @@ def call
 
       raise(TraineeUpdateMissingTrn, "Cannot update trainee on DQT without a trn (id: #{trainee.id})") if trainee.trn.blank?
 
-      dqt_update("/v2/teachers/update/#{trainee.trn}?birthDate=#{trainee.date_of_birth.iso8601}", payload)
+      dqt_update(
+        "/v2/teachers/update/#{trainee.trn}?birthDate=#{trainee.date_of_birth.iso8601}",
+        payload,
+      )
```

Comment:
don't worry in this instance, but generally it'd be good to have formatting changes in a separate commit